tests/migration_test.go

```diff
@@ -1079,6 +1071,8 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				tests.UpdateClusterConfigValueAndWait("migrations", originalMigrationConfig)
 			})
 			It("[test_id:2227]should abort a vmi migration without progress", func() {
+				tests.SkipStressTestIfRunnigOnKindInfraIPv6()
```

Comment:
for kind the stress isnt enough to make the migration fail 
the stress tries to dirty ram pages, in order to fail the migration.
it was approved by Alona and Vladik

Reply:
Then, if the issue happens for all kind providers, why skip it for kind ipv6 ? It is misleading; readers will think it has to do with ipv6 on kind, while it does not. 

I think a comment above the skip with that reasoning (or better yet, have this in an isolated commit with that reasoning in the commit message) is necessary. That is not obvious. At all.

Reply:
even that we have ? (and issue with all the info, this is what Alona asked before)
https://github.com/kubevirt/kubevirt/pull/3221/files#diff-a5a178b4c66835110ceff63c61d28c96R4639

there arent other kind providers that run it / fail on it
so its right to skip for all ? i am not sure
i dont want to close gates in future or past

Reply:
> even that we have ? (and issue with all the info, this is what Alona asked before)
https://github.com/kubevirt/kubevirt/pull/3221/files#diff-a5a178b4c66835110ceff63c61d28c96R4639

Oh, didn't follow through. Since there's an opened issue detailing it, no need to add comment / split commit. Sorry for misleading, should have looked more carefully.

> there arent other kind providers that run it / fail on it
so its right to skip for all ? i am not sure
i dont want to close gates in future or past

I honestly think if the issue happens in all kind providers, it should be skipped in all kind providers. The fact that the test is only executed on the kind ipv6 provider is besides the point, and something people reading the code will not know - unless they go dig the automation script. Which, by the way, is not mandatory to execute tests - I don't use it locally, for instance.

Test code also serves as documentation; whoever reads the test code (and fails to see your issue & read it) will get the feeling the issue is related to ipv6. That's what I'm trying to avoid.

What's the advantage of skipping the test only for kind ipv6, when, according to the [issue](https://github.com/kubevirt/kubevirt/issues/3323), it is orthogonal to all kind providers ? I've stated the disadvantages above.

Reply:
done at 
https://github.com/kubevirt/kubevirt/pull/3379
which will be either pushed here, or we move there
just made an ordered one, and not directly pushed to here,
to see i dont have mistakes